content/3_ModuleThree/31_PartnerSetup.ee.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Partner Setup Instructions" # MODIFY THIS TITLE
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES
+---
+
+# Partner Setup Instructions <!-- MODIFY THIS HEADING -->
```

Comment:
More content guidance.

Reply:
i.e. does the partner require attendees to sign up for things?
https://circleci.awsworkshop.io/ee/040_circleci_setup.html
a la signing up for Docker Hub, Snyk, Terraform etc.